server/routes/viewingLicences/handlers/viewCases.test.ts

```diff
@@ -519,172 +541,118 @@ describe('Route handlers - View and print case list', () => {
-            tabType: 'releasesInNextTwoWorkingDays',
-            nomisLegalStatus: 'SENTENCED',
-            isDueForEarlyRelease: false,
-          },
-          {
-            isClickable: false,
-            licenceId: undefined,
-            licenceStatus: 'IN_PROGRESS',
-            name: 'Harvey Smith',
-            prisonerNumber: 'A1234AC',
-            probationPractitioner: {
-              name: 'Walter White',
-            },
-            releaseDate: '01 May 2022',
-            releaseDateLabel: 'CRD',
-            tabType: 'releasesInNextTwoWorkingDays',
-            nomisLegalStatus: 'SENTENCED',
-            isDueForEarlyRelease: false,
-          },
-          {
-            isClickable: true,
-            licenceId: undefined,
-            licenceStatus: 'SUBMITTED',
-            name: 'Harold Lloyd',
-            prisonerNumber: 'A1234AD',
-            probationPractitioner: {
-              name: 'Harry Goldman',
-            },
-            releaseDate: '01 May 2022',
-            releaseDateLabel: 'CRD',
-            tabType: 'releasesInNextTwoWorkingDays',
-            nomisLegalStatus: 'SENTENCED',
-            isDueForEarlyRelease: false,
-          },
-          {
-            isClickable: true,
-            licenceId: undefined,
-            licenceStatus: 'APPROVED',
-            name: 'Stephen Rowe',
-            prisonerNumber: 'A1234AE',
-            probationPractitioner: {
-              name: 'Larry Johnson',
-            },
-            releaseDate: '10 Jun 2022',
-            releaseDateLabel: 'CRD',
-            tabType: 'futureReleases',
-            nomisLegalStatus: 'SENTENCED',
-            isDueForEarlyRelease: false,
-          },
-        ],
-        ComCreateCaseTab,
-        showAttentionNeededTab: false,
-        hasMultipleCaseloadsInNomis: false,
-        prisonsToDisplay: [
-          {
-            agencyId: 'MDI',
-            description: 'Moorland (HMP)',
-          },
-        ],
-        probationView: false,
-        search: undefined,
-        statusConfig,
-      })
+      expect(res.render).toHaveBeenCalledWith(
+        'pages/view/cases',
+        expect.objectContaining({
+          cases: [
+            expect.objectContaining({
+              prisonerNumber: 'A1234AA',
+              releaseDate: '09 Jan 2022',
+            }),
+            expect.objectContaining({
+              prisonerNumber: 'A1234AB',
+              releaseDate: '01 Apr 2022',
+            }),
+            expect.objectContaining({
+              prisonerNumber: 'A1234AC',
+              releaseDate: '05 Apr 2022',
+            }),
+            expect.objectContaining({
+              prisonerNumber: 'A1234AD',
+              releaseDate: '01 May 2022',
+            }),
+            expect.objectContaining({
+              prisonerNumber: 'A1234AE',
+              releaseDate: '01 Jun 2022',
+            }),
+          ],
+          probationView: false,
+        })
```

Comment:
actual issue was a template problem but whilst looking into this I made the tests more explicit,